src/App.test.js

```diff
@@ -8,6 +8,10 @@ import App from './App';
 import { IntlProvider } from 'react-intl';
 import messages_es from './i18n/es.json';
 import messages_en from './i18n/en.json';
+import jwt_decode from 'jwt-decode';
+
+//Add mock to decode jwt token and not fail
```

Comment:
Very arguably, but I think that these comments are not necessary. What do you think?

Reply:
yes could be